planet.ini

```diff
@@ -1376,3 +1376,11 @@ title = Planet openSUSE
   link     = https://novoslinux.wordpress.com/
   location = en
   author   = irc:tux_
+
+[timojyrinki]
+  title    = SUSE Posts
```

Comment:
I'm not sure if I like this title ;-) - it will make your blog entries look like an "official" SUSE blog (for example, your first post would appear as `SUSE Posts: Package from openSUSE to SLE` on planet.o.o`), but your blog is on a personal domain, and I'd guess that it isn't an official SUSE blog.

I've noticed that you called your blog "SUSE Posts", but nevertheless - are you sure that this title makes sense on planet.opensuse.org? (IMHO another more "personal" title would make more sense, for example your name.)

Reply:
No problem, I'm just feeling very unimaginative regarding the title :) I modified it know to be more clearly unique / mine.